Functions.Templates/Templates/SqlInputBinding-JavaScript/index.js

```diff
@@ -0,0 +1,12 @@
+module.exports = async function (context, results) {
+    if (!!results && results.length > 0) {
+        context.log('Query results: ', JSON.stringify(results));
+    } else {
+        context.log("No results!");
+    }
+    return {
+        status: 201,
+        mimetype:"application/json",
```

Comment:
So I can't seem to find any actual documentation on this and it seems like none of the other templates are setting the type like this so let's just remove this since things seem to work just fine without it. 

(Python has a parameter for this but I'm not actually sure that JS does, or at least I'm not finding it)